lang/cs/Org.Apache.REEF.Wake/Impl/TimerStage.cs

```diff
@@ -40,23 +45,42 @@ public TimerStage(IEventHandler<PeriodicEvent> handler, long period) : this(hand
         /// <param name="period">a period in milli-seconds</param>
         public TimerStage(IEventHandler<PeriodicEvent> handler, long initialDelay, long period)
         {
+            // Core .NET only supports 32 bit timers.e
```

Comment:
Sorry about these garbage characters.  The scroll button on my mouse generates them and I am going to replace it.